packages/admin/src/pages/super-admin-dashboard/user/[id]/change-roles.page.tsx

```diff
@@ -139,4 +135,66 @@ const EditRoleWithId = ({
   );
 };
 
+function renderConditionalCheckboxes(
+  isOwner: boolean,
+  fieldErrors: ValidationError[],
+  roles: Role[],
+  user: User
+) {
+  const adminCheckbox = roles.filter(({ name }) => name === 'ADMIN');
+  const otherCheckboxes = roles.filter(({ name }) => name !== 'ADMIN');
```

Comment:
would be good to use a constant for `ADMIN` instead of magic string - there's probably one in utils somewhere.

Could even also create a function for `isAdmin` and call like
```suggestion
  const isAdmin = ({ name }) => name === CONSTANTS.ADMIN
  const adminCheckbox = roles.filter(isAdmin);
  const otherCheckboxes = roles.filter((role) => !isAdmin(role));
```
but that's probably over the top. Would definitely recommend doing so if we had more conditions here or were doing this check more often, but only two cases means it's fine to do it as you have here

Reply:
redundant with suggested interface change

Reply:
will just leave it here due to number of cases (and no ADMIN constant), although I'll keep it in mind for future when using string literals like this, good to know!